internal/leadership/elector.go

```diff
@@ -13,13 +13,19 @@ import (
 	"github.com/riverqueue/river/internal/notifier"
 	"github.com/riverqueue/river/internal/rivercommon"
 	"github.com/riverqueue/river/internal/util/dbutil"
+	"github.com/riverqueue/river/internal/util/valutil"
 	"github.com/riverqueue/river/riverdriver"
 )
 
 const (
-	electInterval           = 5 * time.Second
-	electInteralJitter      = 1 * time.Second
-	electIntervalTTLPadding = 10 * time.Second
+	electIntervalDefault           = 5 * time.Second
+	electIntervalJitterDefault     = 1 * time.Second
+	electIntervalTTLPaddingDefault = 10 * time.Second
+
+	// TODO: for now we only support a single instance per database/schema.
+	// If we want to provide isolation within a single database/schema,
+	// we'll need to add a client config for this.
+	instanceNameDefault = "default"
```

Comment:
Moved this down from client so it's defined closer to where it's used. Totally fine to move it back up as necessary, but there's no plans AFAIK to make use of it anytime soon and this keeps things clearer for now.

Reply:
kk, hopefully not too big of a problem. Thx.